packages/subgraph/schema.graphql

```diff
@@ -63,6 +63,7 @@ type Proposal @entity {
   weight: BigInt!
   metadata: String
   executedAt: BigInt
+  txHash: String!
```

Comment:
We can't assume the `txHash` won't be empty at the creation time except we initialize it with an empty state. Otherwise when we will do `proposal.save()` in some mapping like the Agreement the subgraph will fail because it expects a non-empty value.